pyproject.toml

```diff
@@ -34,19 +34,43 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
 dynamic = [ "version" ]
+
+#
+# Dependencies and local dev dependencies
+#
+
 dependencies = [
+  "anyio>=4.8",
```

Comment:
I would prefer to leave this as an extra, ie the user must install with `[async]` to support async

Reply:
It's already installed in https internally.